blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpIntegrationTestBase.java

```diff
@@ -65,26 +81,75 @@
     public final String postMultiPath = "/v2.0/%s/ingest/multi";
     public final String postAggregatedPath = "/v2.0/%s/ingest/aggregated";
     public final String postAggregatedMultiPath = "/v2.0/%s/ingest/aggregated/multi";
+
+    public final String getEventsPath = "/v2.0/%s/events/getEvents";
+    public final String getSearchPath = "/v2.0/%s/metrics/search";
+    public final String getTokenSearchPath = "/v2.0/%s/metric_name/search";
+    public final String getViewsPath = "/v2.0/%s/views";
+
     private Random random = new Random( System.currentTimeMillis() );
 
     @BeforeClass
-    public static void setUp() throws Exception{
+    public static void setUpHttp() throws Exception {
+
         Configuration.getInstance().init();
+        Configuration.getInstance().setProperty(CoreConfig.CORS_ALLOWED_ORIGINS, configAllowedOrigins);
+        Configuration.getInstance().setProperty(CoreConfig.CORS_ALLOWED_HEADERS, configAllowedHeaders);
+        Configuration.getInstance().setProperty(CoreConfig.CORS_ALLOWED_METHODS, configAllowedMethods);
+        Configuration.getInstance().setProperty(CoreConfig.CORS_ALLOWED_MAX_AGE, configAllowedMaxAge);
+
+
         // This is to help with Travis, which intermittently fail the following tests due
         // to getting TimeoutException. This is done here because it needs to be before
         // RollupHandler is instantiated.
         Configuration.getInstance().setProperty(CoreConfig.ROLLUP_ON_READ_TIMEOUT_IN_SECONDS, "20");
+
+        setupElasticSearch();
+
+        setupIngestionServer();
+
+        setupQueryServer();
+
+        // setup vendor and client
+        vendor = new HttpClientVendor();
+        client = vendor.getClient();
+    }
+
+    @AfterClass
+    public static void shutdown() {
+        Configuration.getInstance().setProperty(CoreConfig.DISCOVERY_MODULES.name(), "");
+        Configuration.getInstance().setProperty(CoreConfig.ENUMS_DISCOVERY_MODULES.name(), "");
+        Configuration.getInstance().setProperty(CoreConfig.EVENTS_MODULES.name(), "");
+        System.clearProperty(CoreConfig.DISCOVERY_MODULES.name());
+        System.clearProperty(CoreConfig.ENUMS_DISCOVERY_MODULES.name());
+        System.clearProperty(CoreConfig.EVENTS_MODULES.name());
```

Comment:
Calling `Configuration.getInstance().init()` might be simpler instead. It has the added benefit that, if someone in the future changes a different property during a test, they won't have to remember to also reset it. See [here](https://github.com/rackerlabs/blueflood/blob/master/blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java#L83-L86). This makes tests much more independent.